docs/contributing.rst

```diff
@@ -43,12 +43,16 @@ To make a release and **deploy to PyPI**, please follow these steps (we highly s
 
 The flow would then look like follows:
 
-1. :code:`bumpversion (patch|minor)`
-2. :code:`make docs`
-3. :code:`git add CHANGELOG.*`
-4. :code:`git commit -m "update changelog"`
-5. :code:`git push`
-6. :code:`git push --tags`
+1. :code:`bumpversion patch` or :code:`bumpversion minor`
+2. :code:`git describe --tags`, and note this tag name. We will need to edit this tag later.
+3. :code:`make docs`, which will extend the changelog by reading information from github.com/zalando/expan .
+4. :code:`git add CHANGELOG.*`
+5. :code:`git commit --amend --no-edit`
+6. :code:`git show`. Carefully review this commit before proceeding. Ensure the changelog is updated with the expected text, in particular a fully up-to-date version number.
```

Comment:
so if I'm trying to release 1.5.3, I would expect `1.5.3` in changelog?

Reply:
Yes, this works because (as part of the 1.3.1 release) I tweaked the Makefile in order that it uses the correct name of the intended release. See the options `--future-release` in: https://github.com/zalando/expan/blob/852ce9aad809c53316c1fbbbb54d7022b248b24f/Makefile